sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

```diff
@@ -22,22 +22,22 @@ namespace Azure.Core.Pipeline
         private static readonly ConcurrentDictionary<string, object?> ActivitySources = new();
 
         private readonly ActivityAdapter? _activityAdapter;
-        private readonly bool _suppressNestedClientScopes;
+        private readonly bool _suppressNestedClientActivities;
 
-        internal DiagnosticScope(string ns, string scopeName, DiagnosticListener source, ActivityKind kind, bool suppressNestedClientScopes = true) :
-            this(scopeName, source, null, GetActivitySource(ns, scopeName), kind, suppressNestedClientScopes)
+        internal DiagnosticScope(string ns, string scopeName, DiagnosticListener source, ActivityKind kind, bool suppressNestedClientActivities) :
```

Comment:
Curious why we're removing the default here?

Reply:
If we keep default here, it should be consistent with `ClientDiagnostics` and would be suppression *off*.

But the right value for all future manually written clients would be suppression *on*.

If we're ok with having inconsistent default values, I'm happy to return default `= true` back, otherwise, I prefer to force callers to make a choice